kart/gui/diffviewer.py

```diff
@@ -170,7 +174,7 @@ def treeItemChanged(self, current, previous):
         self.widgetDiffConfig.setVisible(True)
         self.tabWidget.setTabEnabled(TAB_GEOMETRY, True)
         self.tabWidget.setTabEnabled(TAB_ATTRIBUTES, True)
-        self.tabWidget.setCurrentIndex(TAB_ATTRIBUTES)
+        self.tabWidget.setCurrentIndex(self.mostRecentTabIndex or TAB_ATTRIBUTES)
```

Comment:
After this we check whether the geometry/attributes tabs should be enabled based on the data... does that have any impact?

I can see `setCurrentIndex()` was called here before though, so presumably not :-)

Reply:
Cleanup in separate commits and/or PRs is all good. Gradually garden it to a better state as we make changes.